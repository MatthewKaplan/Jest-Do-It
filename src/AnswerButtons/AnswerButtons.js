import React, { Component } from 'react';

import '../Styles/_AnswerButtons.scss';

export class AnswerButtons extends Component {

  handleChangeCard = () => {
    this.props.checkAnswer(this.props.answer);
  }

  render() {
    const cardAnswer = this.props.answer;
    return (
      <div className="answerButtons">
        <button onClick={this.handleChangeCard}>{cardAnswer}</button>
      </div>
    )
  }
}


export default AnswerButtons;
