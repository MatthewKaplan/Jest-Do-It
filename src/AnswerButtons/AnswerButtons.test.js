import React from 'react';
import { AnswerButtons } from './AnswerButtons';
import { shallow } from 'enzyme';

describe('AnswerButtons', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<AnswerButtons data={{}} />);
    expect(wrapper).toMatchSnapshot();
  });
});