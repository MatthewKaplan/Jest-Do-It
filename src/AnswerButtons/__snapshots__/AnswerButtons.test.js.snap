// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnswerButtons should match snapshot 1`] = `
<div
  className="answerButtons"
>
  <button
    onClick={[Function]}
  />
</div>
`;
